src/styles/stars.css

```diff
@@ -0,0 +1,37 @@
+.stars {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+  background-image: url(../images/star.svg);
+}
+
+.stars__star:last-child {
+  margin: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url(../images/star-active.svg);
```

Comment:
If you reuse the same css-rule, you can do like this

.stars--1 .stars__star:nth-child(-n + 1),
.stars--2 .stars__star:nth-child(-n + 2,
.stars--3 .stars__star:nth-child(-n + 3),
.stars--4 .stars__star:nth-child(-n + 4),
.stars--5 .stars__star:nth-child(-n + 5) {
  background-image: url(../images/star-active.svg);
}